test/Microsoft.Azure.Devices.Edge.Test/SetupFixture.cs

```diff
@@ -132,21 +132,9 @@ public async Task BeforeAllAsync()
                     await this.daemon.UninstallAsync(token);
 
                     // Delete test certs, keys, etc.
-                    Directory.Delete(FixedPaths.E2E_TEST_DIR, true);
-
-                    // Restore backed up config files.
-                    foreach ((string file, string _) in this.configFiles)
```

Comment:
this code is no longer in mater. Confirmed with @gordonwang0 what it should be removed from release/1.2 as well.